app/Http/Requests/Teams/UpdateTeamRequest.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\Http\Requests\Teams;
+
+use Illuminate\Foundation\Http\FormRequest;
+use Illuminate\Validation\Rule;
+
+class UpdateTeamRequest extends FormRequest
+{
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules(): array
+    {
+        return [
+            'name' => [
+                'required',
+                'min:3',
+                'max:100',
+                Rule::unique('teams')->ignore($this->get('id'))
+            ],
+            'identifier' => [
+                'required',
+                'min:3',
+                'max:15',
+                Rule::unique('teams')->ignore($this->get('id'))
```

Comment:
Why are we ignoring the ID here? The comment block should explain this. Not immediately clear for the reader

Reply:
You can see the full explanation here https://laravel.com/docs/8.x/validation#rule-unique, on the section `Forcing A Unique Rule To Ignore A Given ID`. This is a very common rule actually.